.github/workflows/test-deploy.yml

```diff
@@ -120,7 +120,13 @@ jobs:
 
           echo ::set-output name=tags::"$(
             for registry in docker.io/{dvcorg,iterativeai} ghcr.io/iterative; do
-              [[ "${{ matrix.latest }}" == "true" ]] && echo "${registry}/cml:latest"
+              if [[ "${{ matrix.latest }}" == "true" ]]; then
+                if [[ "${{ matrix.gpu }}" == "true" ]]; then
+                  echo "${registry}/cml:latest-gpu"
+                else
+                  echo "${registry}/cml:latest"
+                fi
+              fi
```

Comment:
```suggestion
              [[ "${{ matrix.latest }}${{ matrix.gpu }}" == "truetrue" ]] && echo "${registry}/cml:latest"
```

Reply:
I originally wanted to have only `latest` with implicit GPU support, but don't know if it's a good idea.